spin.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+
+set -e
+
+if [[ "$(uname)" != "Darwin" ]]; then
+  echo "This script can run only on OS-X"
+fi
+
+env=${1:-'dev'}
+
+function self_signed_certificate() {
+    if [[ -f web_s2i/cert.pem && -f web_s2i/key.pem ]]; then
+      return
+    fi
+    openssl req -x509 -newkey rsa:4096 -nodes -out web_s2i/cert.pem \
+      -keyout web_s2i/key.pem -days 365 \
+      -subj "/C=IL/ST=Gush-Dan/L=Tel-Aviv/O=DevOps Loft/OU=''/CN=''"
+}
+
+self_signed_certificate
+if [[ "$env" == "dev" ]]; then
+  vagrant box update --provider virtualbox
+  secure_port=8443
+else
+  secure_port=443
+fi
+vagrant destroy -f $env
+vagrant up $env
+server_name=$(vagrant ssh-config $env | grep HostName | awk \{'print $2'\})
+sleep 5
+"/Applications/Google Chrome.app/Contents/MacOS/Google Chrome" \
```

Comment:
What are those for?
and won't they only work on OSx?

Reply:
This PR is still WIP